dspace/config/modules/authentication.cfg

```diff
@@ -24,6 +24,10 @@
 #  * X.509 Certificate Authentication
 #        Plugin class: org.dspace.authenticate.X509Authentication
 #        Configuration file: authentication-x509.cfg
+
+# ORCID certificate authentication.
+# plugin.sequence.org.dspace.authenticate.AuthenticationMethod = org.dspace.authenticate.OrcidAuthenticationDelegator
```

Comment:
Class not present in branch, should probably be `org.dspace.authenticate.OrcidAuthentication` to enabled ORCID authentication